terraform-modules/aws/amplify/main.tf

```diff
@@ -0,0 +1,63 @@
+resource "aws_iam_role" "amplify" {
```

Comment:
Should add in the `tags` key and pass in all of our standard tags

Reply:
updated